src/applications/personalization/dashboard-2/components/DashboardWidgetWrapper.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function DashboardWidgetWrapper({ children }) {
```

Comment:
@sanlouise I think you can use this in the health care section. One would wrap the "next appointment" and another would wrap the "health care links".

Reply:
Thank you! Hey, an old school `function` 😄

Reply:
Not too opinionated, but I would have maybe called it something like `<GridWrapper />` or `<LayoutWrapper />` so it's pretty clear why we've added this wrapper

Reply:
Hmm... I think that `DashboardWidget` should be in there, since it'll be used to wrap multiple widgets on the dashboard. Oh I can use it for Applications in Progress, too! Want to call it `DashboardWidgetLayoutWrapper`?

Reply:
Haha, that's way too long of a component name IMO. Keep it as is, I'm good with it! 👍